Docker.md

```diff
@@ -4,7 +4,7 @@
 
 - [Docker](https://docs.docker.com/get-docker/) (Docker Desktop or on Linux normal Docker)
 - [Docker Compose](https://docs.docker.com/compose/install/) (Included in Docker Desktop)
-- LEGO® Universe Client (packed or unpacked). Check the main [README](./README.md) for details on this.
+- LEGO® Universe Client packed client. Check the main [README](./README.md) for details on this.
```

Comment:
Maybe change the wording for the time being to: `LEGO® Universe packed Client.`

Reply:
Done